proofOfConcept/ColorPickerAlpha/ColorPickerAlpha/MainWindow.xaml.cs

```diff
@@ -11,21 +12,52 @@ public partial class MainWindow : Window
     {
         Boolean rgbState = false;
         Color curColor;
+        OverlayWindow overlayWnd;
+        private bool _pickerActive = true;
+        public bool pickerActive
+        {
+            get { return _pickerActive; }
+            set 
+            {
+                _pickerActive = value;
+                overlayWnd.Visibility = value ? Visibility.Visible : Visibility.Hidden;
+                Topmost = false;
+                Topmost = true;
```

Comment:
Topmost can't be both false and true. Unless you got Schrodinger's cat

Reply:
Its an issue I found online where sometimes it gets finicky if not set to false first

Reply:
https://stackoverflow.com/questions/3729369/topmost-is-not-topmost-always-wpf

Reply:
looool